examples/layer-browser/src/data-samples.js

```diff
@@ -109,6 +109,17 @@ export const zigzag = [
   }
 ];
 
+export const greatCircles = [
+  {source: [-20, 0], target: [-160, 0]},
+  {source: [-20, 10], target: [-160, 10]},
+  {source: [-20, 20], target: [-160, 20]},
+  {source: [-20, 30], target: [-160, 30]},
+  {source: [-20, 40], target: [-160, 40]},
+  {source: [-20, 50], target: [-160, 50]},
+  {source: [-20, 60], target: [-160, 60]},
+  {source: [-20, 70], target: [-160, 70]}
+];
```

Comment:
Need more test cases. E.g. longitude 10 to longitude -170. Source and target not on the same latitude.

Reply:
more test cases are added.
![great_circle_example](https://user-images.githubusercontent.com/42359372/52580304-edf7b600-2ddc-11e9-961c-d55204121a8f.png)